src/main.rs

```diff
@@ -58,6 +58,7 @@ pub fn main() -> std::io::Result<()> {
     opts.optflag("d", "dry-run", "dry run, produce no output");
     opts.optflag("e", "events", "print event sequence instead of rendering");
     opts.optflag("T", "enable-tables", "enable GitHub-style tables");
+    opts.optflag("M", "enable-math", "enable LaTeX-style math");
```

Comment:
This clashes with `enable-metadata-blocks`, defined below. Running out of letters :(